packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/l10n/network_lookup/error_lookup.dart

```diff
@@ -21,8 +21,6 @@ class AppI18nErrorLookup extends I18nErrorLookup {
           bundledI18n = I18nError(I18nErrorLookup_bg());
           break;
         case 'en':
-          bundledI18n = I18nError(I18nErrorLookup_en());
```

Comment:
Please make sure such changes are addressed in separate PRs. From what I see this is not related to QR code integration, but feel free to correct me if I'm wrong